src/CurrentIdentity/Storage/MemoryCurrentIdentityStorage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\User\CurrentIdentity\Storage;
+
+/**
+ * Stores current identity ID in memory.
+ */
+final class MemoryCurrentIdentityStorage implements CurrentIdentityStorageInterface
```

Comment:
Is it used anywhere?

Reply:
Authentication has to be performed on each request, which is often the case for stateless application such as RESTful API.